core/src/main/java/io/kestra/core/services/FlowService.java

```diff
@@ -53,6 +54,9 @@ public class FlowService {
     @Inject
     ApplicationContext applicationContext;
 
+    @Value("${kestra.system-flows.namespace:system}")
```

Comment:
I supposed that the syntax `:system` is used to set a default, but I think for user it would be great to put it in the application.yml, so its more explicit

Reply:
Yes, it provides a default, we use this syntax already in a few places and it avoids adding it to the various configuration files we have.

>  for user it would be great to put it in the application.yml, so its more explicit

What we really need is an example config file with all configuration point and comments. I already thought about it. But this is for a different PR ;)